nvidia-driver.spec

```diff
@@ -4,15 +4,15 @@
 %if 0%{?rhel} == 6
 # RHEL 6 does not have _udevrulesdir defined
 %global _udevrulesdir   %{_prefix}/lib/udev/rules.d/
-%global _dracutopts     nouveau.modeset=0 rdblacklist=nouveau
-%global _dracutopts_rm  nomodeset vga=normal
+%global _dracutopts     rdblacklist=nouveau
+%global _dracutopts_rm  nomodeset vga=normal nouveau.modeset=0
```

Comment:
Further down in the SPEC file I just check for the whole string of options, and the other two have been there for quite a while, so I guess all users will not have those and the `nouveau.modeset=0` parameter would not get removed.

I guess I need to remove the old Dracut options and just leave that, or iterate on each of one in the SPEC file.